### ISSUE TITLE
Error when launching program

### ISSUE THREAD

**Issue by flip111** [mpue/blackboard#15]
```
Error in Java launcher. Error occured while calling main method in Java class org/pmedv/blackboard/app/BlackBoard
```

On system Windows 7 64 bits. Java (JRE) 8 update 45 (build 1.4.5_45-b15).

**Comment by kazakh87** [comment]
I have same problem.
The BlackBoardSetup_1.1.3.exe for Windows in install dir.
Windows 10 64 bit
Java JDK 1.8.0_91

**Comment by Lasermax3000** [comment]
I´ve got the same issue by starting the BlackBoard.exe file. 
Is there a solution yet, since this issue has been closed? 
My System Win 11 with latest Java SE Development Kit 20.

**Comment by dcoredump** [comment]
Same problem with `Kubuntu-22.04.3 LTS` and `blackboard-bin-2.0.0.2`

Fix for me:
```
mkdir -p $HOME/.BlackBoard/simulators
mkdir -p $HOME/.BlackBoard/models
mkdir -p $HOME/.BlackBoard/symbols
```

**Comment by kebakaran007** [comment]
Windows 11, Java SE 11, had the same issue.
Fixed by copying blackboard-bin-2.0.0.2\models, \parts, \simulators, \symbols
to
C:\Users\USER\.BlackBoard.

**Comment by KyryIx** [comment]
I use with Windows 7 and 11.
I used idea of @kebakaran007.
In my situation is:
instaled in: C:\BlackBoard
directories models, parts, simulators, symbols in: C:\Users\everton\.BlackBoard
log in: C:\Users\everton\.BlackBoard\log
